### PR TITLE
Incorporate documentation feedback for Ceph Monitoring and MDS

### PR DESCRIPTION
**Pull request by fmount**

### CODE REVIEW

**Review by klgill — CHANGES_REQUESTED** [review]
1 minor comment.

File: docs_user/modules/proc_migrating-ceph-mds.adoc [hunk 1]
```diff
@@ -202,14 +197,17 @@ It is possible to elect a dedicated MDS as "active" for a particular file system
 When failing over MDS daemons, cluster monitors prefer standby daemons with `mds_join_fs` equal to the file system name with the failed rank. If no standby exists with `mds_join_fs` equal to the file system name, it chooses an unqualified standby as a replacement.
 +
 ----
-$ ceph config set mds.mds.cephstorage-0.fqcshx mds_join_fs cephfs
+$ sudo cephadm shell -- ceph config set mds.mds.cephstorage-0.fqcshx mds_join_fs cephfs
 ----
 
+- Replace mds.mds.cephstorage-0.fqcshx with the daemon deployed on
+  cephstorage-0 retrieved from the previous step
```
[the remark this answers is not in the document]
Comment (fmount) [reply docs_user/modules/proc_migrating-ceph-mds.adoc]: Done, thank you for the quick review!

**Comment by fmount** [comment]
and rebased.

**Review by klgill — APPROVED** [review]

**Review by jistr — APPROVED** [review]

**Comment by jistr** [comment]
recheck

**Pull request merged**